Packs/epo/Integrations/epoV2/README.md

```diff
@@ -0,0 +1,1563 @@
+
+
+### epo-query-table
+***
+Queries an ePO table.
+
+
+#### Base Command
+
+`epo-query-table`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| target | Table name. | Required | 
+| select | The columns to select, in SQUID syntax. Example: "(select EPOEvents.AutoID EPOEvents.DetectedUTC EPOEvents.ReceivedUTC)". | Optional | 
+| where | Filter results, in SQUID syntax. Example: "(where ( eq ( OrionTaskLogTask .UserName "ga" )))". | Optional | 
+| order | Order in which to return the results, in SQUID syntax. Example: "(order (asc OrionTaskLogTask.StartDate) )"). | Optional | 
+| group | Group the results, in SQUID Syntax. Example: "(group EPOBranchNode.NodeName)". | Optional | 
+| joinTables | Perform join, in SQUID syntax. | Optional | 
+| query_name | Name for the query to appear in the context. | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| McAfee.ePO.Query | unknown | Query result. | 
+
+
+#### Command Example
+```!epo-query-table target="FW_Rule"```
+
+#### Context Example
+```json
+{
+    "McAfee": {
+        "ePO": {
+            "Query": {
+                "1639695731": {
+                    "6544666": [
```

Comment:
we should not have these random numbers in the context.
And I don't see here the query name as part of the output

Reply:
For the name part there is an argument to overwrite these number and we I did  one more change to do it with the target